R/ebook.R

```diff
@@ -15,16 +15,17 @@
 #' @param epub_version Whether to use version 3 or 2 of EPUB.
 #' @param md_extensions A character string of Pandoc Markdown extensions.
 #' @param pandoc_args A vector of additional Pandoc arguments.
+#' @inheritParams html_chapters
```

Comment:
I prefer explicitly documenting this argument, and let users know the default EPUB templates can be found in the repo https://github.com/jgm/pandoc-templates